modules/flake-parts/packages.holochain-tx5.nix

```diff
@@ -0,0 +1,45 @@
+{
+  # System independent arguments.
+  lib,
+  inputs,
+  ...
+}: {
+  perSystem = {
+    # Arguments specific to the `perSystem` context.
+    self',
+    pkgs,
+    ...
+  }: {
+    # system specific outputs like, apps, checks, packages
+
+    packages = let
+      system = pkgs.system;
+      craneLib = inputs.crane.lib.${system};
+      cranePkgs = inputs.crane.inputs.nixpkgs.legacyPackages.${system};
+    in {
+      tx5 = craneLib.buildPackage {
+        pname = "tx5";
+        src = inputs.tx5;
+        version = inputs.tx5.rev;
+        cargoExtraArgs = "--examples --bins";
+        nativeBuildInputs = [
+          cranePkgs.perl
+          cranePkgs.pkg-config
+          cranePkgs.go
+        ];
+
+        doCheck = false;
+      };
+
+      tx5-signal-srv = pkgs.runCommandNoCC "tx5-signal-srv" {} ''
+        mkdir -p $out/bin
+        cp ${self'.packages.tx5}/bin/tx5-signal-srv $out/bin/
```

Comment:
i used the tx5 package can be used to access `turn-stress` and `turn_doctor`, and other potentially useful binaries.

Reply:
That still sounds like extra compilation to me, I'd prefer to be specific and it's always possible to build more or everything later

Reply:
i put in some optimization for this but i'm not going to 100% because i don't see it as a worthwhile investment at this point.

specifically it gets complicated trying to pick binaries in the dependency derivation:

```
$ nix build .#tx5-signal-srv
warning: Git tree '/home/steveej/src/holo/holochain-infra' is dirty
error: builder for '/nix/store/wdv39a08nqz6z87v8622nkp8szn8mr0x-tx5-deps-d5fa9096eae2156618c701e87b7c40770faa123c.drv' failed with exit code 101;
       last 10 log lines:
       >     crane-dummy-tx5
       >     crane-dummy-tx5-core
       >     crane-dummy-tx5-demo
       >     crane-dummy-tx5-go-pion
       >     crane-dummy-tx5-go-pion-sys
       >     crane-dummy-tx5-go-pion-turn
       >     crane-dummy-tx5-online
       >     crane-dummy-tx5-signal
       >     crane-dummy-tx5-signal-srv
       >
       For full logs, run 'nix log /nix/store/wdv39a08nqz6z87v8622nkp8szn8mr0x-tx5-deps-d5fa9096eae2156618c701e87b7c40770faa123c.drv'.
```

the commit i pushed compiles all dependencies and then feeds them into the `tx5` (all examples, all binaries) and the `tx5-signal-srv` (only single binary) derivations.